cmd/spicedb/schemawatch_integration_test.go

```diff
@@ -0,0 +1,127 @@
+//go:build docker && image
+// +build docker,image
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/ory/dockertest/v3"
+	"github.com/ory/dockertest/v3/docker"
+	"github.com/stretchr/testify/require"
+
+	testdatastore "github.com/authzed/spicedb/internal/testserver/datastore"
+	"github.com/authzed/spicedb/pkg/datastore"
+)
+
+func TestSchemaWatch(t *testing.T) {
+	engines := map[string]bool{
+		"postgres":    false,
+		"mysql":       false,
+		"cockroachdb": true,
+		"spanner":     false,
```

Comment:
did you mean to disable postgres and spanner?

Reply:
Yes